source/includes/_ethereum.md

```diff
@@ -422,12 +512,12 @@ It might occur that a callback function of a sent query gets called more than on
 
 ### Encrypted Queries
 Certain contexts, such as smart contracts on public blockchains, might require a level of privacy to protect data from public scrutiny. Developers can make encrypted Oraclize queries by encrypting a part (or all) of a query with the Oraclize public key.
-The encrypted queries feature may be of interested to developers who want to deploy their blockchain applications of public networks. For example, if an application leverages data from an authenticated API, it would be dangerous to disclose the API key to anyway who is monitoring the public chain.  
+The encrypted queries feature may be of interested to developers who want to deploy their blockchain applications of public networks. For example, if an application leverages data from an authenticated API, it would be dangerous to disclose the API key to anyway who is monitoring the public chain.
```

Comment:
```suggestion
The encrypted queries feature may be of interest to developers who want to deploy their blockchain applications on public networks. For example, if an application leverages data from an authenticated API, it may be dangerous to disclose the API key to anybody who is monitoring the public chain.
```

Reply:
pssht, and English is your first and only language, come on Grgorsz

Reply:
Shh!! I already feel inadequate in an office full of polyglots!!

Also, how on earth do you pronounce `Grgorsz`?